service/awsconfig/v5/cloudconfig/master_template.go

```diff
@@ -31,7 +31,7 @@ func (c *CloudConfig) NewMasterTemplate(customObject v1alpha1.AWSConfig, certs l
 			customObject: customObject,
 			keys:         keys,
 		}
-		params.Hyperkube.Apiserver.Docker.CommandExtraArgs = c.k8sAPIExtraArgs
+		params.Hyperkube.Apiserver.Pod.CommandExtraArgs = c.k8sAPIExtraArgs
```

Comment:
@corest After vendoring k8scloudconfig I needed to update to fix a compile error. Is the new path correct?